portfolio/src/main/webapp/kite_gallery.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html id = "gallery">
+  <head>
+    <meta charset="UTF-8">
+    <title>Kite Image Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="gallery_content">
+      <h1 id = "g_title">My Kite Image Gallery</h1>
+      <div class="image-thumbnails">
```

Comment:
that's a great example of how to implement https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns
good